backend/lib/models/IndividualUser.js

```diff
@@ -0,0 +1,79 @@
+const { Schema } = require("mongoose");
+const { model: User } = require("./User");
+const { updateAuthorName: updatePostAuthorName } = require("./Post");
+const { updateAuthorName: updateCommentAuthorName } = require("./Comment");
```

Comment:
Where are this methods define?
They are not at `Post` and `Comment` at master and from what I understood you didn't add them at this PR.

Reply:
Originally added on [this commit](https://github.com/FightPandemics/FightPandemics/pull/401/commits/e39b358d1ad91fca7af3735f6df53a046747e345) but I think the merge deleted them 🤔